media/css/externalpages/pocket/components/_add.scss

```diff
@@ -0,0 +1,169 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '../utils/variables';
+@import './platform-nav';
+@import './platform-footer';
+
+$image-path: '/media/img/externalpages/pocket';
+
+.add-page {
+    .add-header {
+        background-color: $color-background-grey;
+    }
+
+    h2 {
+        color: $color-text-dark-grey;
+        font-size: $size200;
+        margin-bottom: $spacing100;
+    }
+
+    h3,
+    p {
+        color: $color-text-grey;
+    }
+}
+
+.add-header {
+    &__wrapper {
```

Comment:
Yeah I naturally went with BEM notation when we first coded the pages, happy to change them, but just wanted to get pages up and running since we'll be redoing much of the designs in 2022 Q1 anyway. I'll add it to the `.stylelintignore` file